README.md

```diff
@@ -54,6 +54,21 @@ In the data format, every timeseries is described by six dimensions (codes):
 6.	Subannual (optional, default 'Year')<sup>[1]</sup> -
     [more information](subannual)
 
+In addition to these six dimensions, every timeseries is described by
```

Comment:
time-series

Reply:
It seems that there is no clear preference between *timeseries*, *time series* and *time-series*, and I don't see much difference in readability (in contrast to sub-annual where I have used the hyphen-version in text but not in the column name). 

For our use case, the term is mostly used as *timeseries data* where *time series data* looks a bit weird (to me at least) - so I have until now tried to always use the one-word spelling (also in the [pyam docs](https://pyam-iamc.readthedocs.io)). So I'd prefer to keep it as is for consistency.

Objections?

Reply:
No, It's ok for me.